inlang/source-code/sdk/src/adapter/solidAdapter.test.ts

```diff
@@ -278,6 +350,73 @@ describe("messages", () => {
 			).value
 		).toBe("test2")
 	})
+
+	it("should react to message delete", async () => {
+		const repo = await mockRepo()
+		const fs = repo.nodeishFs
+		await fs.mkdir("/user/project.inlang.inlang", { recursive: true })
```

Comment:
the tests seem ill-placed. 

the solid adapter acts as reactivity bridge. the only tests that are required is briding reactivity, not testing business logic. the added tests (and old tests too btw) seem to test business logic.

Reply:
Will remove the delete test from here. They don't go very deep anyway.
Deletion in now tested in loadProject.test and covers similar ground.